prs/WeatherData.cs

```diff
@@ -0,0 +1,35 @@
+namespace Sample {
+    /**
+     * A class which updates displays with various atmospheric data.
+     */
+    public class WeatherData {
+
+        private float temperature;
```

Comment:
nit: private variables in C# should start with "_". For example` private float _temperature`.